specification/cognitiveservices/OpenAI.Inference/models/images/images.tsp

```diff
@@ -179,7 +179,13 @@ model ImageGenerations {
 // Note: pending resolution of  cross-language code emission behavior for long-running operations, image generation
 //        reuses its final operation response model as its status polling model.
 
-// @lroStatus
+// Note2: the Java emitter currently does not support versioning annotations
```

Comment:
Since all SDK is preview/beta, I kind of think that all deprecated API should be removed from SDK upon its first GA release. Upon GA, SDK should only contain the API that service wish user to call.

Though in current beta (and maybe next few iterators of beta), if we wish to not break customer (and giving them time to move from the old API to new API), we can include the old LRO API in client.tsp. In Java code, we can mark the API as "deprecated" and recommend user to use the new API.
(user can pin the [ServiceVersion](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIServiceVersion.java) to e.g. V2023_09_01_PREVIEW)